client/src/components/SpendingInsights.jsx

```diff
@@ -44,7 +44,7 @@ export default function SpendingInsights({ transactions }) {
         : (obj[tx.name] = tx.amount);
       return obj;
     }, {});
-  }, [transactions]);
+  }, [monthlyTransactions]);
 
   // sort names by spending totals
   const sortedNames = useMemo(() => {
```

Comment:
I know this might have not been introduced in this PR, but `sortedNames.map()` is created a React error in the console:

![Screen Shot 2021-06-14 at 10 19 01 PM](https://user-images.githubusercontent.com/516877/121997051-88c27f80-cd5e-11eb-9f51-f2dea8d69c67.png)